tests/functional/test_api_versions.py

```diff
@@ -49,3 +49,8 @@ def test_command_interface_reflects_api_version(self):
         cmdline = 'ec2 describe-nat-gateways'
         _, stderr, _ = self.run_cmd(cmdline, expected_rc=2)
         self.assertIn("Invalid choice: 'describe-nat-gateways'", stderr)
+
+    def test_handles_ProfileNotFound_exception(self):
+        self.environ['AWS_PROFILE'] = 'foo'
+        # ProfileNotFound exception shouldn't be raised
+        self.run_cmd('--version')
```

Comment:
It would be nice to explicitly add `expected_rc=0` to this command to point out the assertion is that the command succeeds.